src/Components/PresentationalComponents/CVEPageDetailsDescription/CVEPageDetailsDescription.js

```diff
@@ -120,9 +121,11 @@ const CVEPageDetailsDescription = ({ cveAttributes }) => {
                     </Flex>
                 </StackItem>
                 <StackItem>
-                    <TextContent>
-                        <Text component={TextVariants.p}>{description}</Text>
-                    </TextContent>
+                    <TruncatedText
+                        component={TextVariants.p}
```

Comment:
@leSamo actually we don't want the `TruncatedText` for the CVE description. (we might use it in the future) but based on the mockups we'll need the component for another long text  .